server/Makefile

```diff
@@ -705,7 +705,7 @@ config-openid: ## Configures OpenID.
 config-reset: ## Resets the config/config.json file to the default production values.
 	@echo Resetting configuration to production default
 	rm -f config/config.json
-	OUTPUT_CONFIG=$(PWD)/config/config.json $(GO) $(GOFLAGS) run -tags production ./scripts/config_generator
+	OUTPUT_CONFIG=$(PWD)/config/config.json $(GO) run $(GOFLAGS) -tags production ./scripts/config_generator
```

Comment:
Previously, if `GOFLAGS` was set, this would fail. (We didn't previously have a reason to set `GOFLAGS`).